app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,63 @@
+class OrdersController < ApplicationController
+  include OrdersHelper
+  include SessionsHelper
+  ORDER_PARAMS = %i(place payment_method user_name user_phone).freeze
+  before_action :set_default_data, only: %i(order_info create)
+  before_action :set_order_items_ids, only: %i(order_info create)
+
+  def order_info
+    @order = Order.new
+  end
+
+  def create
+    @order = Order.new order_params
+    @order.user_id = @current_user.id
```

Comment:
```suggestion
    @order = @current_user.orders.new order_params
```
chữ học ở chapter 13-14 rụng hết rồi à em